docs/Product-Strategy/Roadmap-and-Backlog.md

```diff
@@ -45,10 +41,11 @@ _Risks:_ The users that would be on-boarded to this process, would potentially h
 | Files successfully transferred to ACF server | Not Started
 | Files uploaded by STT user(s) are successfully entered into the existing TDRS system | Not Started
 
-## Release 3 (fka STT MVP)
-Value Delivered: Create a database that will parse data, communicate error messages to the user that are easier to read and act upon. The scope of this release is still being defined, but the team agrees that the priority for focusing on processing data is the next right step.
+## Release 3: Parity, Data Parsing, Automated Status, Notifications
+Value Delivered: Create a database that will parse data, communicate error messages to the user that are easier to read and act upon. This release will allow us to onboard users and eventually depreciate the TANF Data Reporting System.
```

Comment:
Just a terminology tweak here: I'd say "Create a software system with a relational database that will...."